app/controllers/users_controller.rb

```diff
@@ -1,6 +1,8 @@
 class UsersController < ApplicationController
+  before_action :authenticate_user!
   def index
     @users = User.all
+    # @user = current_user
```

Comment:
- [ ] Please kindly delete the commented line of code to keep your code clean and look more professional 😁